# Patch necessary to solve a linker error on Fedora 30
# for `major` and `minor` symbols.
# Lars Viklund, Sun 18 Aug 2019 01:31:21 AM CEST
diff -ru numactl-2.0.11.orig/affinity.c numactl-2.0.11/affinity.c
--- numactl-2.0.11.orig/affinity.c	2015-12-10 22:47:05.000000000 +0100
+++ numactl-2.0.11/affinity.c	2019-08-18 01:29:47.999886806 +0200
@@ -39,6 +39,7 @@
 #include <dirent.h>
 #include <linux/rtnetlink.h>
 #include <linux/netlink.h>
+#include <sys/sysmacros.h>
 #include <sys/types.h>
 #include <ctype.h>
 #include <assert.h>
